Kernel/Arch/aarch64/MMU.cpp

```diff
@@ -214,10 +214,18 @@ static void activate_mmu()
     Aarch64::TCR_EL1::write(tcr_el1);
 
     // Enable MMU in the system control register
-    Aarch64::SCTLR_EL1 sctlr_el1 = Aarch64::SCTLR_EL1::read();
+    Aarch64::SCTLR_EL1 sctlr_el1 = {};
```

Comment:
There are other flags here that default to being set. We shouldn't be clearing those!

Reply:
There's other flags as well that (seem to) default to "uninitialized" or architecturally unknown values. Unless I read the documentation incorrectly... Do you know which exact flags we should have set?

Reply:
> SA0, bit [4]

> SP Alignment check enable for EL0. When set to 1, if a load or store instruction executed at EL0 uses the SP as the base address and the SP is not aligned to a 16-byte boundary, then an SP alignment fault exception is generated. For more information, see SP alignment checking on page D1-5387.
> When FEAT_VHE is implemented, and the value of HCR_EL2.{E2H, TGE} is {1, 1}, this bit has no effect on execution at EL0.
> The reset behavior of this field is:
> • On a Warm reset, this field resets to an architecturally UNKNOWN value

Reply:
All right, I'm using the reset_value() helper on the SCTLR_EL1 register, and I updated it for Rev J.a of the Arm ARM